asv_bench/benchmarks/rolling.py

```diff
@@ -68,9 +69,21 @@ class Quantile(object):
     param_names = ['constructor', 'window', 'dtype', 'percentile']
 
     def setup(self, constructor, window, dtype, percentile):
-        N = 10**5
+        N = 10 ** 5
         arr = np.random.random(N).astype(dtype)
         self.roll = getattr(pd, constructor)(arr).rolling(window)
 
     def time_quantile(self, constructor, window, dtype, percentile):
         self.roll.quantile(percentile)
+
+    def time_quantile_nearest(self, constructor, window, dtype, percentile):
```

Comment:
Ha when I asked for updated ASVs I was just asking for a refresh of what you posted earlier in the thread but I suppose this is better. Is it possible to just add interpolation as an argument to `setup`? Would be more concise.

Here's an example from GroupBy that you could use for reference, though the existing Quantile class could clue you in on how to do this as well:

https://github.com/pandas-dev/pandas/blob/78fee04e95e3c53c83c938285580c39e7761ddc8/asv_bench/benchmarks/groupby.py#L342

Reply:
Please also post results to a comment when done. I'm assuming that these will fail on master but would still be good to know how they baseline against the current implementation that you posted earlier in thread